src/Pmi/Controller/BiobankController.php

```diff
@@ -253,6 +254,38 @@ public function ordersTodayAction(Application $app, Request $request)
         ]);
     }
 
+    public function quanumOrdersTodayAction(Application $app, Request $request)
+    {
+        // Get beginning of today (at midnight) in user's timezone
+        $startString = 'today';
+        // Allow overriding start time to test in non-prod environments
+        if (!$app->isProd() && intval($request->query->get('days')) > 0) {
+            $startString = '-' . intval($request->query->get('days')) . ' days';
+        }
+        $startTime = new \DateTime($startString, new \DateTimeZone($app->getUserTimezone()));
+        // Get MySQL date/time string in UTC
+        $startTime->setTimezone(new \DateTimezone('UTC'));
+        $today = $startTime->format('Y-m-d');
+
+        $quanumOrders = $app['pmi.drc.participants']->getOrders([
+            'startDate' => $today,
+            'endDate' => date('Y-m-d'),
```

Comment:
Summarizing conversation on Slack - because the RDR endpoint only filters on date and not date/time, we can't actually accomplish the goal of showing today's orders for the user's timezone. What we probably want to do is make sure that the end date is today in UTC (the latest possible date since we're only concerned about happenings in the western hemisphere) and use the start date of the user's time zone. This will get extra orders in some circumstances, but I think is better than missing some orders that should be considered "today".

Reply:
Resolved with 07075bd